openrouteservice/src/main/java/org/heigit/ors/api/util/SystemMessage.java

```diff
@@ -0,0 +1,293 @@
+package org.heigit.ors.api.util;
```

Comment:
In general, would it be better to keep the core handling of generating system messages outside of the api package? One of the main ideas of having the API in it's own package was that all the code did there was to convert information between an API request and the object needed by the ors core - anything more than this conversion should happen within the main code itself. So in my mind, it would be better to move the system message class out of the api package and into the heigit.ors.util package. Then it keeps the system messages a bit more independent from the api itself which may be better for if ors gets used as a library and then the API code could effectively be ignored.

Reply:
I really don't think so, since the system messages are in my mind part of the API and independent from the core functionalities. If someone were to use ORS as lib and not use our API code, they'd also have no use for the system messages code.

Reply:
As far as I understood, the messages are meant to inform the API-users about changes in the services such as the deprecation of the old API, of parameters or of profiles. All these examples are unrelated to the core, hence, the core would be the wrong place for these use cases. Do we have any use cases, where the core would like to inject such a message in the response?

Further, the class imports several classes from the API, which is inappropriate for a class in the core and would be a serious design flaw.

However, we can also question, whether the API would be the right place. All of the above examples are messages about the services we run and not necessarily about the functions provided by the API. So we could even argue to place the messages at an even more abstract level. However, with the current setup, I believe that putting them into the API would be easier.

Reply:
OK